urlimport.go

```diff
@@ -0,0 +1,201 @@
+package main
+
+import "fmt"
+import "flag"
+import "net/http"
+import "os"
+import "io"
+import "io/ioutil"
+import "net/url"
+import "strings"
+import "time"
+import "net"
+
+import "github.com/jlaffaye/ftp"
+
+var filename string
+var downloadURL string
+var log *os.File
+var logfileName string
```

Comment:
This is another minor thing, but the variables defined here could be combined into a single `var` statement as well.